languages/go/exercises/concept/basic-slices/basic_slices.go

```diff
@@ -0,0 +1,36 @@
+package basic_slices
+
+// GetItem retrieves an item from a slice at given position. The second return value indicates whether
+// a the given index existed in the slice or not.
+func GetItem(slice []uint8, index int) (uint8, bool) {
+	panic("not implemented")
```

Comment:
I would include the name of the function to be implemented here, similar to https://github.com/exercism/research_experiment_1/blob/master/exercises/go-1-a/go_1_a.go#L5.

Reply:
Can do :)

Note: Although a `panic` will create a stack trace so it is quite obvious where it is coming from.

Reply:
OK.